MicroTumorLinux1_Amazon_Setup-README.md

```diff
@@ -0,0 +1,113 @@
+To install a new dashboard on Ubuntu 14.04, run the following commands:
+
+```
+sudo apt-get update
+sudo apt-get upgrade
+sudo apt-get install git make gcc g++ libncurses5-dev subversion libssl-dev libx11-dev libxt-dev libgl1-mesa-dev libosmesa6-dev libglu1-mesa-dev libfontconfig-dev libxrender-dev ninja-build -y
+sudo apt-get install python-numpy numpy-scipy python-pip -y
+sudo apt-get install libhdf5-dev -y # required by 'tables'
+sudo pip install IPython tornado zmq jinja2 tables matplotlib
+```
+
+Install cmake
+
+```
+mkdir ~/Support
+cd ~/Support
+wget http://www.cmake.org/files/v3.5/cmake-3.5.0.tar.gz
+tar xzvf cmake-3.5.0.tar.gz
+cd cmake-3.5.0
+./configure
+make -j4
+cd ..
+mv cmake-3.5.0 cmake-3.5.0-nossl
+tar -xvzf cmake-3.5.0.tar.gz
+mkdir cmake-3.5.0-build
+cd cmake-3.5.0-build/
+../cmake-3.5.0-nossl/bin/cmake -DCMAKE_USE_OPENSSL:BOOL=ON -DCMAKE_USE_OPENSSL:BOOL=ON -DCMAKE_BUILD_
+TYPE:STRING=Release ../cmake-3.5.0
+make -j4
+sudo make install
+```
+
+Install Qt4.8.7 (https://github.com/jcfr/qt-easy-build.git)
```

Comment:
Instead of duplicating the code. Could you consider adding a `linux_build_qt.sh` to qt-easy-build ?